fresque/lib/views.py

```diff
@@ -73,10 +78,20 @@ def newpackage(db, method, data, username):
             db.commit()
         except SQLAlchemyError:
             result.flash.append(("An error occurred while adding your "
-                 "package, please contact an administrator.", "danger"))
+                "package, please contact an administrator.", "danger"))
         else:
             result.flash.append(("Package successfully created!", "success"))
             result.redirect = ('package', {"name": pkg.name})
+
+            # Add git repo creation message
+            try:
+                status_message = create_git_repo(pkg.name, gitfolder)
+                result.flash.append((
+                    status_message, "success"))
+            except (GitError, IOError):
```

Comment:
Well the IOError may not be a big error, it could be that it's just that the repo already exists.

Might be worth using your own exception class rather than IOError to figure out what's going on